scripts/storage-emulator-integration/tests.ts

```diff
@@ -1710,6 +1721,63 @@ describe("Storage emulator", () => {
             items: [],
           });
         });
+
+        context("with folder placeholders", () => {
```

Comment:
can you include a test that validates that invalid paths are filtered out of the list response please, e.g. `//foo/` `bar//` `/baz//qux/`

Reply:
Done.